test/select.jl

```diff
@@ -819,6 +819,49 @@ end
     @test_throws ArgumentError select(sdf, :x1 => identity => :r1, copycols=false)
 end
 
+@testset "pseudo-broadcasting" begin
+    df = DataFrame([1 2 3
+                    4 5 6])
+    df2 = DataFrame([1 2 3])
+    df3 = DataFrame(x1=Int[], x2=Int[], x3=Int[])
+    for v in [9, Ref(9), view([9], 1)]
+        @test select(df, [] => (() -> 9) => :a, :, (:) => (+) => :d) ==
```

Comment:
I think you wanted to use `v` instead of `9`, right?

Reply:
sure. now you see how I developed these tests :). Corrected